src/util.js

```diff
@@ -81,6 +81,10 @@ exports.buildOptions = function(options, defaultOptions, props) {
   return newOptions;
 };
 
+const nameStartChar = ':A-Za-z_\\u00C0-\\u00D6\\u00D8-\\u00F6\\u00F8-\\u02FF\\u0370-\\u037D\\u037F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD';
+const nameChar = nameStartChar + '\\-.\\d\\u00B7\\u0300-\\u036F\\u203F-\\u2040';
+exports.nameRegexp = '[' + nameStartChar + '][' + nameChar + ']*'
+
```

Comment:
Can it be a good idea of defining the range of letters that should not be included in the tag name? Because any other Unicode character should be allowed. Moreover, there can be more ranges I believe.

Reply:
Which other ranges? I took the values from xml spec.

Reply:
It was my doubt. But I tested this for randomly for 15 languages and they're covered. So it seems fine. Now, my concern is performance. Have you checked the impact on performance when by default all the ranges are applicable?

Reply:
Looks like this change *improves* performance. I ran with and without the change - each commit twice, and picked the better result.

Before:
```
validation : 12234.247180941054 requests/second
xml to json : 10602.1257025917 requests/second
xml to json + json string : 7859.281152576787 requests/second
xml to json string : 1747.9156403661627 requests/second
xml2js  : 2985.894748422982 requests/second
```

After:
```
validation : 13801.418604604587 requests/second
xml to json : 13016.920442592424 requests/second
xml to json + json string : 10186.32443980399 requests/second
xml to json string : 1807.3535925877125 requests/second
xml2js  : 3310.3600184283914 requests/second
```

Reply:
Thanks for the update. I'll probably test on my machine too.